pages/credit-card.tsx

```diff
@@ -13,6 +14,7 @@ export default function CreditCard(props) {
         DCWordProblem, //debit card word problem
         CCDCMulti, //multiple choice
         CardColorProblem, //credit debit both color problem
+        TestComponent,
```

Comment:
lets not have this extra stage. I think this assignment is good enough based on what is in the teacher's PDF. I would focus on completing those requirements before adding in new stuff. 

If you're going to add new screens, I'd want to see you draw it out on a piece of paper before attempting to build it. I used to do this all the time as a junior dev, but it's really easy to immediately want to try building something without knowing exactly how it's going to work. 

If you want to add your own stuff to this lesson, draw out what it'll look like first. Otherwise just stick to what the teacher has in her assignment